### DISCUSSION TITLE
Clarify "Core.SkipPim": true setting

### DISCUSSION THREAD

**Discussion by serhatVIRT** [Azure/AzOps#823] [Q&A] [answered]
Hi guys! I have an question about the "Core.SkipPim": true  setting with the following description: Do not include Privileged Identity Management resources in pull. 

But is not really clear for me what this settings does. Can someone maybe clarify this settings what it does? I tested with the values true and false but didn't notice anything at all. 

Thanks in advance!

**Comment by daltondhcp** [comment, marked as answer]
Sorry for the late reply. It pulls down PIM roleAssignments, the resource type roleEligibilityScheduleRequests. 
See further details in https://github.com/Azure/AzOps/pull/635
